src/EnergyPlus/ChillerGasAbsorption.cc

```diff
@@ -978,16 +981,7 @@ void GasAbsorberSpecs::initialize(EnergyPlusData &state)
     Real64 rho = 0.0;  // local fluid density
     Real64 mdot = 0.0; // lcoal fluid mass flow rate
 
-    if (this->oneTimeFlag) {
-        this->setupOutputVariables(state);
-        this->oneTimeFlag = false;
-    }
-
     // Init more variables
-    if (this->plantScanFlag) {
-        this->oneTimeInit(state);
```

Comment:
Deleted the this->oneTimeInit calls, since oneTimeInit_new will be called from the plantManager.